common/src/address/mod.rs

```diff
@@ -44,8 +44,7 @@ impl Address {
         })
     }
 
-    #[allow(dead_code)]
-    pub(crate) fn new_with_hrp<T: AsRef<[u8]>>(hrp: &str, data: T) -> Result<Self, AddressError> {
+    pub fn new_with_hrp<T: AsRef<[u8]>>(hrp: &str, data: T) -> Result<Self, AddressError> {
```

Comment:
This is not supposed to be pub. You're supposed to use the chain config version from above. But anyway, I can't see the usage of this anywhere.

Reply:
At some point I thought I needed to use this function in the mempool tests.
Changed back to pub(crate).